src/esp/bindings_js/modules/web_demo.js

```diff
@@ -50,19 +50,30 @@ class WebDemo {
       scope: document.getElementById("scope"),
       status: document.getElementById("status")
     });
+
+    this.task.init();
+    this.task.reset();
   }
 
   updateAgentConfigWithSensors(agentConfig = defaultAgentConfig) {
     const sensorConfigs = [
       {
         uuid: "rgb",
+        sensorType: Module.SensorType.COLOR
+      },
+      {
+        uuid: "left_eye",
         sensorType: Module.SensorType.COLOR,
-        resolution: [480, 640]
+        resolution: [1024, 1024]
```

Comment:
Hold on. The position of these two sensors should be offset by the VR headsets (interpupilary distance (IPD) / 2) (or the average IPD for the population if not provided. And doesn't the rgb sensor become redundant with this setup. No reason to waste render cycles on it.

Reply:
I originally had a position with IPD set here, but it just gets overridden by the transforms coming from WebXR every frame anyway.

Reply:
See the stereo example agent: https://github.com/facebookresearch/habitat-sim/blob/b6faca590ccb52a672074c0b14b8d7dd47b4d3de/examples/tutorials/stereo_agent.py

Reply:
Ah I see, I would still have it set as a default.